modules/rostests/rosautotest/main.cpp

```diff
@@ -87,6 +89,15 @@ extern "C" int
 wmain(int argc, wchar_t* argv[])
 {
     int ReturnValue = 1;
+    DWORD TestStartTime, TestEndTime;
+
+    GetModuleFileNameW(NULL, TestName, _countof(TestName));
+//    printf("Full TestName is '%S'\n", TestName);
+    WCHAR* Name = wcsrchr(TestName, '\\');
+    if (Name)
+        Name++;
+    memmove(TestName, Name, _countof(TestName) * sizeof(WCHAR));
```

Comment:
```suggestion
    if (Name)
        memmove(TestName, Name + 1, _countof(TestName) * sizeof(WCHAR));
```
The reasoning: 1. Don't call `memmove` in case `Name` was NULL, and 2. If for whatever reason there isn't a backslash, just use whatever is in the TestName buffer (we cannot do much at this point).